storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/Changes.java

```diff
@@ -19,13 +21,37 @@
     private final Set<String> affectedTables;
 
     /**
+     * Immutable set of affected tags.
+     */
+    @NonNull
+    private final Set<String> affectedTags;
```

Comment:
I thought I will be less convenient for the most frequent case with `Strings`. 
But I agree tags it is not about common cases, so flexibility should be the main purpose.
@rsinukov also votes for `object`.
Let's do it=)